update-and-commit-translations.sh

```diff
@@ -5,7 +5,7 @@ set -e
 set -u
 set -x
 
-tx pull -f --mode=reviewed
+tx pull -f -a --minimum-perc=1 --traceback
```

Comment:
Per the [docs](https://docs.transifex.com/client/debug-and-traceback#traceback), it looks like `--traceback` should be the first argument?

Reply:
```suggestion
tx --traceback pull -f -a --minimum-perc=1
```